lib/litani_report.py

```diff
@@ -392,15 +392,16 @@ async def release_html_dir(args):
 
 
 async def acquire_html_dir(args):
-    remaining = args.timeout
+    start = datetime.datetime.now()
+    timeout = args.timeout or 10 ** 10    # >300 years
+    end = start + datetime.timedelta(seconds=timeout)
```

Comment:
300 years is probably a sensible default, but why not
```python
end = datetime.datetime.max
if args.timeout:
    end = start + datetime.timedelta(seconds=timeout)
```
Just feels like the cleaner way of doing it

Reply:
I didn't know about `datetime.datetime.max`, thank you!